tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1950,6 +1950,29 @@ def feature_importance(self, data=None, model=None, features=None, feature_stage
             fi_df[low_str] = pd.Series(ci_low_dict)
         return fi_df
 
+    def compile_models(self, compiler_configs={}):
+        """
+        Compile models for accelerated prediction.
+        This can be helpful to reduce prediction latency and improve throughput.
+
+        Note that this is currently an experimental feature, the supported alternative compiler can be ['native', 'onnx'].
+
+        Parameters
+        ----------
+        compiler_configs : dict, default = {}
+            compiler : str, default = 'native'
+                The compiler that is used for model compilation.
+            batch_size : int, default = None
```

Comment:
unused?

Reply:
Yes, this is currently unused for onnx backend. As this is going to be a backend that would be used for other backends, the batch_size argument is very much useful to backends that doesn't support dynamic shape. Examples could be:
* https://onnxruntime.ai/docs/tutorials/mobile/helpers/make-dynamic-shape-fixed.html